examples/MultiXcodeProject Example/Podfile

```diff
@@ -0,0 +1,24 @@
+workspace 'Examples.xcworkspace'
+platform :ios, '9.0'
+
+install! 'cocoapods',
+	 generate_multiple_pod_projects: true
```

Comment:
Why not always generate multiple pod projects?  Isn't it recommended to deintegrate anyway before doing a CocoaPods version update?

Reply:
I personally haven't heard about deintegrating before a version update, but we didn't want to turn it on for everyone by default since it's a pretty significant change and didn't want to potentially break projects. In future versions, this could very much be turned on by default.

Reply:
agreed let's keep this as an option for now and later version we can switch the default _or_ in a 2.x version remove the single project generator entirely.